src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -467,6 +479,8 @@ private Task ProcessStreamItem(HubConnectionContext connection, StreamItemMessag
 
         streamCts ??= CancellationTokenSource.CreateLinkedTokenSource(connection.ConnectionAborted);
 
+        var activity = CreateActivity(scope.ServiceProvider, descriptor.MethodExecutor.MethodInfo.Name);
```

Comment:
The method name passed in here is always the .NET name. Can the method name be changed with an attribute? It should be the logical name.